docs/getting-started/run-superbench.md

```diff
@@ -0,0 +1,33 @@
+---
+id: run-superbench
+---
+
+# Run SuperBench
+
+Having prepared benchmark configuration and inventory files,
```

Comment:
What's the meaning of this sentence?

Reply:
[this one](https://ell.stackexchange.com/questions/57471/the-meaning-of-having-done-my-homework-i-will-go-home#:~:text=Having%20done%2FHaving%20finished%20is,carry%20out%20the%20second%20action.)?